### ISSUE TITLE
Webpages out of date

### ISSUE THREAD

**Issue by dagewa** [dials/dials#1066]
Various parts of https://dials.github.io do not reflect the state of play in late 2019.
![Bayeux](https://user-images.githubusercontent.com/11502083/70390328-e6377c80-19c1-11ea-9166-2f027494886d.jpg)

>The DIALS framework is being developed

> We have begun by reproducing published spot finding and integration algorithms used by integration packages such as XDS [2] and MOSFLM

> we hold regular meetings, with talks from invited speakers, and code camps 

Add @benjaminhwilliams and @noemifrisina to DIALS East

[Dr Nicholas Sauter](http://pbd.lbl.gov/scientists/nicholas-sauter/) Link broken

> Developers include ... Iris Young

Acknowledgements should point to https://github.com/dials/dials/blob/master/AUTHORS

**Comment by dagewa** [comment]
In general, the whole page might be too wordy (?)

**Comment by dagewa** [comment]
This may no longer be correct
https://dials.github.io/missing-images.html#refinement

**Comment by dagewa** [comment]
@jbeilstenedmands Should this also have a `symmetry` run _after_ scaling, for sys abs check?
https://dials.github.io/dials_scale_user_guide.html#advanced-use-choosing-reflections-to-use-for-minimisation

**Comment by dagewa** [comment]
https://dials.github.io/license.html != https://github.com/dials/dials/blob/master/LICENSE

**Comment by dagewa** [comment]
The https://dials.github.io/links.html page is mostly useless

**Comment by Anthchirp** [comment]
> https://dials.github.io/license.html != https://github.com/dials/dials/blob/master/LICENSE

suggest just linking to https://github.com/dials/dials/blob/master/LICENSE directly instead of having a separate license page.

**Comment by dagewa** [comment]
:+1: have done, and similar for AUTHORS

**Comment by dagewa** [comment]
Well, they're not linked, but imported directly into the page when the website is built. I think that's the cleanest solution

**Comment by dagewa** [comment]
@phyy-nx @nksauter Can I update the Berkeley lab logo? It does not look like we have the current one.
https://creative.lbl.gov/downloads/the-berkeley-lab-brand/

**Comment by nksauter** [comment]
Yes
Nicholas K. Sauter, Ph. D.
Senior Scientist, Molecular Biophysics & Integrated Bioimaging Division
Lawrence Berkeley National Laboratory
1 Cyclotron Rd., Bldg. 33R0345
Berkeley, CA 94720
(510) 486-5713
